utils/fixed_decimal/src/decimal.rs

```diff
@@ -799,7 +817,7 @@ impl FixedDecimal {
     /// assert_eq!("-0.0", dec.to_string());
     /// ```
     pub fn ceil(&mut self, position: i16) {
-        if self.is_negative {
+        if self.sign == Sign::Negative {
```

Comment:
Suggestion: it may be useful to have a new function `.is_negative()` that returns a bool.

Reply:
Given that this isn’t a general-purpose numeric type (cf. my comment on `negate`), that the sign is discarded for pluralization, and that `Negative` should behave like `Positive` for formatting, it’s not clear to me that we need that.